docs/community/hosting.md

```diff
@@ -0,0 +1,28 @@
+# Hosting New Projects on GradleUp
+
+New Gradle related projects can be potentially hosted on GradleUp.
+It includes transferring the project to GradleUp,
+and ensuring quality and sustainability of the project.
+
+## Hosting Process
+
+At the moment, there is no documented process for hosting new components.
+The criteria below are general practices,
+and they will be extended over time.
+The decision will be made by the [GradleUp Team](./team.md) on a case by case basis.
+
+To request the transfer and initiate the discussion, contact the team.
+
+## Criteria
+
+The general criteria for hosting a project are:
+
+- The project has value to the ecosystem and fits into the GradleUp scope.
+- The project must be open source and licensed under one of the
+  [OSI-approved open source licenses](https://opensource.org/),
+  preferably the Apache License v2.
```

Comment:
Out of curiosity, any reason to prefer Apache vs MIT?

Reply:
My personal preference is always MIT or BSD, but Apache v2's patent grant is supposedly very useful for safeguarding against legal actions

Reply:
Yep, Apache License prevents from patent claims. Will remove the specifics for now though